internal/pkg/deploy/deploy.go

```diff
@@ -127,6 +132,54 @@ func (s *Store) listDeployedWorkloads(appName string, envName string, workloadTy
 	return wklds, nil
 }
 
+// ListDeployedSNSTopics returns a list of SNS topics deployed to the current environment and tagged with
+// Copilot identifiers.
+func (s *Store) ListDeployedSNSTopics(appName string, envName string) ([]Topic, error) {
```

Comment:
nit: I think since this is the `deploy` pkg and we take as input the app and env names we can remove "Deployed" from the name
```suggestion
func (s *Store) ListSNSTopics(appName, envName string) ([]Topic, error) {
```